tests/Unit/Domain/CoordinateMaps/TestMapHelpers.hpp

```diff
@@ -176,45 +180,52 @@ void test_coordinate_map_argument_types(
                       Map::dim>(unwrapped_array);
   };
 
-  const auto mapped_point = map(test_point);
-  CHECK_ITERABLE_APPROX(map(add_reference_wrapper(test_point)), mapped_point);
-  CHECK_ITERABLE_APPROX(map(make_array_data_vector(test_point)),
+  const auto mapped_point = map(test_point, args...);
+  CHECK_ITERABLE_APPROX(map(add_reference_wrapper(test_point), args...),
+                        mapped_point);
+  CHECK_ITERABLE_APPROX(map(make_array_data_vector(test_point), args...),
                         make_array_data_vector(mapped_point));
   CHECK_ITERABLE_APPROX(
-      map(add_reference_wrapper(make_array_data_vector(test_point))),
+      map(add_reference_wrapper(make_array_data_vector(test_point)), args...),
       make_array_data_vector(mapped_point));
 
   if (CheckInverse) {
-    const auto expected = map.inverse(mapped_point);
-    CHECK_ITERABLE_APPROX(map.inverse(add_reference_wrapper(mapped_point)),
-                          expected);
-    CHECK_ITERABLE_APPROX(map.inverse(make_array_data_vector(mapped_point)),
-                          make_array_data_vector(expected));
-    CHECK_ITERABLE_APPROX(map.inverse(add_reference_wrapper(
-                              make_array_data_vector(mapped_point))),
-                          make_array_data_vector(expected));
+    const auto expected = map.inverse(mapped_point, args...);
+    CHECK_ITERABLE_APPROX(
+        map.inverse(add_reference_wrapper(mapped_point), args...), expected);
+    CHECK_ITERABLE_APPROX(
+        map.inverse(make_array_data_vector(mapped_point), args...),
+        make_array_data_vector(expected));
+    CHECK_ITERABLE_APPROX(
+        map.inverse(add_reference_wrapper(make_array_data_vector(mapped_point)),
+                    args...),
+        make_array_data_vector(expected));
   }
 
   {
-    const auto expected = map.jacobian(test_point);
-    CHECK_ITERABLE_APPROX(map.jacobian(add_reference_wrapper(test_point)),
-                          expected);
-    CHECK_ITERABLE_APPROX(map.jacobian(make_array_data_vector(test_point)),
-                          make_tensor_data_vector(expected));
+    const auto expected = map.jacobian(test_point, args...);
```

Comment:
make one lambda which has variadic arguments and contains all the CHECKS; then have an overloader that calls this lambda with the appropriate arguments